gradle/checkstyle.gradle

```diff
@@ -3,6 +3,7 @@ apply plugin: 'checkstyle'
 checkstyle.toolVersion = '7.8.2'
 checkstyle.configFile = rootProject.file('config/checkstyle/checkstyle.xml')
 checkstyleMain.exclude '**/BuildConfig.java'
+checkstyleTest.enabled = false
```

Comment:
We can still check the code style to keep ourselves in check :)

Reply:
It's only for test code. On the other hand it was failing even before this PR because tests don't have any javadoc on public methods and so on.

Reply:
Yeah, let's keep checkstyle as it was (if possible).

Reply:
It works just because we use '_checkstyleMain_' task on CI and it definitely won't work if use '_checkstyle_'